src/v1/_base.scss

```diff
@@ -31,12 +31,19 @@
         bottom: 0;
         opacity: 0;
 
-        &[aria-hidden="true"] {
+        &[aria-hidden='true'] {
           visibility: hidden;
         }
       }
     }
   }
+
+  // New features are not implemented in theme-v1, corresponding HTML code is hidden
+  .str-chat__unread-messages-notification,
+  .str-chat__unread-messages-separator-wrapper,
+  .str-chat__message-bounce-prompt {
```

Comment:
The message bounce prompt doesn't have a selector in React, so this is Angular-only